### PR TITLE
chore(deps): update to tauri beta.9

### PR DESCRIPTION
**Pull request by lucasfernog**

### CODE REVIEW

File: examples/api/src-tauri/Cargo.toml [hunk 1]
```diff
@@ -54,4 +60,4 @@ tauri-plugin-biometric = { path = "../../../plugins/biometric/", version = "2.0.
 window-shadows = "0.2"
 
 [features]
-custom-protocol = [ "tauri/custom-protocol" ]
+prod = ["tauri/custom-protocol"]
```
Comment (amrbashir) [inline examples/api/src-tauri/Cargo.toml:63]: do we still need this here? (some other examples have it as well)

File: plugins/deep-link/examples/app/src-tauri/Cargo.toml [hunk 2]
```diff
@@ -26,4 +26,4 @@ tauri-plugin-deep-link = { path = "../../../" }
 # this feature is used for production builds or when `devUrl` points to the filesystem and the built-in dev server is disabled.
 # If you use cargo directly instead of tauri's cli you can use this feature flag to switch between tauri's `dev` and `build` modes.
 # DO NOT REMOVE!!
-custom-protocol = [ "tauri/custom-protocol" ]
+prod = ["tauri/custom-protocol"]
```
Comment (amrbashir) [inline plugins/deep-link/examples/app/src-tauri/Cargo.toml:29]: ditto

File: plugins/single-instance/examples/vanilla/src-tauri/Cargo.toml [hunk 3]
```diff
@@ -17,4 +17,4 @@ tauri-plugin-single-instance = { path = "../../../" }
 tauri-build = { workspace = true }
 
 [features]
-custom-protocol = [ "tauri/custom-protocol" ]
+prod = ["tauri/custom-protocol"]
```
Comment (amrbashir) [inline plugins/single-instance/examples/vanilla/src-tauri/Cargo.toml:20]: ditto

File: plugins/updater/tests/app-updater/Cargo.toml [hunk 4]
```diff
@@ -15,4 +15,4 @@ tiny_http = "0.11"
 time = { version = "0.3", features = ["formatting"] }
 
 [features]
-custom-protocol = ["tauri/custom-protocol"]
+prod = ["tauri/custom-protocol"]
```
Comment (amrbashir) [inline plugins/updater/tests/app-updater/Cargo.toml:18]: ditto

**Review by amrbashir — DISMISSED** [review]

**Comment by lucasfernog** [comment]
i wonder why the updater key is broken

**Pull request merged**